testkit-backend/pom.xml

```diff
@@ -32,6 +32,24 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We try to avoid Lombok…

Reply:
I could delombok the code, it's not an issue. But the `testkit-backend` is not a library, it's an application.